kernel/kernel.asm

```diff
@@ -2474,8 +2470,7 @@ endg
 iglobal
 version_inf:
         db 0,7,7,0  ; version 0.7.7.0
-        db 0
-.rev    dd __REV__
+.rev    file "rev.inc"
```

Comment:
Please only include 40 bytes of the file here (`file "rev.inc":0,40`) and create the "rev.inc" using a regular shell redirect.